crates/sdk/src/masp/utils.rs

```diff
@@ -58,17 +104,64 @@ pub trait MaspClient<'client, C: Client> {
         progress: &impl ProgressTracker<IO>,
         shutdown_signal: &mut ShutdownSignal,
         tx_sender: FetchQueueSender,
-        from: u64,
-        to: u64,
+        from: BlockHeight,
+        to: BlockHeight,
     ) -> Result<(), Error>;
+
+    /// Return the capabilities of this client.
+    #[inline(always)]
+    fn capabilities(&self) -> MaspClientCapabilities {
```

Comment:
Do we need these default implementations?

Reply:
No, this can be refactored.